### PR TITLE
feat: initial setup

### PR DESCRIPTION
**Pull request by FredrikOseberg**
This PR sets up the feature flag for the new strategy configuration and duplicates the components for the new setup

### CODE REVIEW

File: frontend/src/component/feature/FeatureStrategy/NewFeatureStrategyCreate/NewFeatureStrategyCreate.tsx [hunk 1]
```diff
@@ -0,0 +1,249 @@
+import React, { useEffect, useRef, useState } from "react";
```
Comment (FredrikOseberg) [inline frontend/src/component/feature/FeatureStrategy/NewFeatureStrategyCreate/NewFeatureStrategyCreate.tsx:1]: Duplicated

File: frontend/src/component/feature/FeatureStrategy/NewFeatureStrategyEdit/NewFeatureStrategyEdit.test.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { formatUpdateStrategyApiCode } from "component/feature/FeatureStrategy/FeatureStrategyEdit/FeatureStrategyEdit";
```
Comment (FredrikOseberg) [inline frontend/src/component/feature/FeatureStrategy/NewFeatureStrategyEdit/NewFeatureStrategyEdit.test.tsx:1]: Duplicated

File: frontend/src/component/feature/FeatureStrategy/NewFeatureStrategyEdit/NewFeatureStrategyEdit.tsx [hunk 3]
```diff
@@ -0,0 +1,308 @@
+import { useEffect, useRef, useState } from "react";
```
Comment (FredrikOseberg) [inline frontend/src/component/feature/FeatureStrategy/NewFeatureStrategyEdit/NewFeatureStrategyEdit.tsx:1]: Duplicated

**Review by kwasniew — APPROVED** [review]
Only reviewed the flag setup and flag usage. I trust the copy paste :)

**Pull request merged**